data/json/construction.json

```diff
@@ -2224,5 +2224,17 @@
     "components": [ [ [ "pillow", 19 ], [ "down_pillow", 19 ] ], [ [ "down_blanket", 3 ], [ "blanket", 3 ], [ "fur_blanket", 3 ] ] ],
     "pre_terrain": "t_floor",
     "post_terrain": "f_pillow_fort"
+  },
+  {
+    "type": "construction",
+    "skill": "survival",
+    "description": "Build Fire Ring",
+    "category": "FURN",
+    "difficulty": 1,
```

Comment:
I think it might be better to set it to use "required_skills" format, like other construction entries use (like pillow fort directly above)